tempesta_fw/sock.c

```diff
@@ -482,15 +485,39 @@ ss_tcp_process_data(struct sock *sk)
 		skb_dst_drop(skb);
```

Comment:
Shouldn't this be done in the egress path, i.e. in `ss_do_send()`? Until that time it doesn't matter at all as the SKB is taken out of the Linux TCP/IP stack and orphaned.